Classes/Helper/BITHockeyHelper.m

```diff
@@ -199,53 +200,81 @@ void bit_fixBackupAttributeForURL(NSURL *directoryURL) {
   return settingsDir;
 }
 
-#pragma mark - Keychain
+#pragma mark - UserDefaults
 
-BOOL bit_addStringValueToKeychain(NSString *stringValue, NSString *key) {
-  if (!key || !stringValue)
-    return NO;
-  
-  NSString *serviceName = [NSString stringWithFormat:@"%@.BugsplatMac", [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleIdentifier"]];
-  
-  BITGenericKeychainItem *item = [BITGenericKeychainItem genericKeychainItemForService:serviceName withUsername:key];
-  
-  if (item) {
-    // update
-    [item setPassword:stringValue];
-    return YES;
-  } else {
-    if ([BITGenericKeychainItem addGenericKeychainItemForService:serviceName withUsername:key password:stringValue])
-      return YES;
-  }
-  
-  return NO;
+NSString *getUserDefaultsKey(NSString* key)
```

Comment:
Why doesn't this name follow the same pattern the others follow?

Reply:
The thought around this is that this is an internal function